scripts/build/cardano-sl.sh

```diff
@@ -192,6 +192,9 @@ if [[ $clean == true ]]; then
   echo "Cleaning cardano-sl-wallet"
   stack clean cardano-sl-wallet
 
+  echo "Cleaning cardano-sl-wallet-new"
+  stack clean cardano-sl-wallet-new
```

Comment:
One more change is needed in this script. There are lines like these:
```
  elif [[ $var == "wallet" ]]; then
    spec_prj="wallet"
  elif [[ $var == "explorer" ]]; then
    spec_prj="explorer"
```

You should add `wallet_new` case there.
P. S. Yes, these scripts are stupid and a bit tedious to maintain, because it's bash and we don't have bash experts unfortunately :(

Reply:
Ok, makes sense -- maybe we want to build cardano with just `wallet-new` or similar. Leave it with me.